pkg/controller/data_upload_controller.go

```diff
@@ -430,6 +442,29 @@ func (r *DataUploadReconciler) findDataUploadForPod(podObj client.Object) []reco
 	return []reconcile.Request{requests}
 }
 
+func (r *DataUploadReconciler) FindDataUploads(ctx context.Context, cli client.Client, ns string) ([]velerov2alpha1api.DataUpload, error) {
+	pods := &corev1.PodList{}
+	var dataUploads []velerov2alpha1api.DataUpload
+	if err := cli.List(ctx, pods, &client.ListOptions{Namespace: ns}); err != nil {
+		r.logger.WithError(errors.WithStack(err)).Error("failed to list pods on current node")
+		return nil, errors.Wrapf(err, "failed to list pods on current node")
+	}
+
+	for _, pod := range pods.Items {
+		if pod.Spec.NodeName != r.nodeName {
```

Comment:
The same as above, add a comment and error log

Reply:
Added